### PR TITLE
feat: add golden signals pack

### PR DESCRIPTION
**Pull request by aswanson-nr**
## Summary

A pack to supply base alert conditions and a dashboard for web applications and APIs.

dashboard

### CODE REVIEW

File: packs/golden-signals-web/alerts/errors.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+
+name: Errors
+
+details: |+
+  This alert fires when the 10 percent of the transactions against an application end with an error, over a period of 5 minutes.
```
Comment (polfliet) [inline packs/golden-signals-web/alerts/errors.yml:6]: typo: remove `the`

File: packs/golden-signals-web/alerts/memory.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+
+name: Memory Usage
+
+details: |+
+  When memory limits are reached, applications can do weird and unpredicable things.
```
Comment (polfliet) [inline packs/golden-signals-web/alerts/memory.yml:6]: typo: `unpredictable`

File: packs/golden-signals-web/alerts/response_time.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+
+name: Response time
+
+details: |+
+  This alert fires when the average transaction duration is above 5 seconds for 5 mintues.
```
Comment (polfliet) [inline packs/golden-signals-web/alerts/response_time.yml:6]: typo: `minutes`

File: packs/golden-signals-web/config.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+# Description of the pack
+description: |+
+  Golden signals alerts for web servers. Includes alerts and a dashboard for throughput, errors, response time, CPU usage, and memory usage.
+
+# Support level: New Relic | Verified | Community
+level: New Relic
+
+# Design
+icon: icon.jpeg
```
Comment (polfliet) [inline packs/golden-signals-web/config.yml:13]: I guess there isn't really a better icon/logo for this, so I am ok with the New Relic logo

**Review by polfliet — COMMENTED** [review]
Great contribution Alec!
I just noticed a few typos that need to be fixed, and then it would be great to merged this in!

**Comment by rudouglas** [comment]
Fixed all typos @polfliet

**Review by polfliet — APPROVED** [review]

**Pull request merged**